### PR TITLE
make API as a separate directory

### PR DESCRIPTION
**Pull request by amarts**

### CODE REVIEW

**Comment by amarts** [comment]
@Vikastc with this, I am not able to get the UI for swagger. Can you check this out? The command to run the scripts etc are given already.

**Comment by Vikastc** [comment]
Will do

**Comment by amarts** [comment]
actually, i just replaced this apis.json in another repo where docs is working, and looks like everything in json file is fine. Mostly a express server issue, may be.

File: api/index.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import fs from "fs";
+import swaggerUi from "swagger-ui-express";
+import express from 'express';
+import http from 'http';
+
+const { PORT } = process.env;
+
+const openApiDocumentation = JSON.parse(
+  fs.readFileSync("./apis.json").toString()
```
Comment (Vikastc) [inline api/index.ts:9]: Should this be ./api/apis.json?

File: api/index.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+);
+
+console.log("Test", openApiDocumentation);
+const app = express();
+app.get("/docs", swaggerUi.serve, swaggerUi.setup(openApiDocumentation));
```
Comment (Vikastc) [inline api/index.ts:14]: This should be app.use, 
think that is the problem

Comment (Vikastc) [reply api/index.ts:14]: @amarts

Comment (amarts) [reply api/index.ts:14]: I tried both, and still seeing the issue.

Comment (Vikastc) [reply api/index.ts:14]: I got it when i tried

Comment (amarts) [reply api/index.ts]: Ack. You are right! using `app.use()` instead of app.get() it is working. Sending the fix.

**Review by Vikastc — APPROVED** [review]

**Pull request merged**